docker/download-aosp.sh

```diff
@@ -27,12 +27,15 @@ function parse_opts() {
 					ARCH=$2
 					case "${ARCH}" in
 						x86)
+							TARGET='aosp_x86-trunk_staging-userdebug'
```

Comment:
Please double check that these are the right targets: The original ones had `cf` and `phone` in them, but these don't.

It's also strange that the target for arm64 is a cf phone, but those for intel are not even cuttlefish.

Reply:
Thanks. Updated x86-64 target as cf target.
But I noticed that there is no cf target for x86 . 
What should be the proper target for x86?

Reply:
@jemoreira PTAL